.changeset/hot-paws-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/coin-tron": patch
```

Comment:
This is not a `patch` as it doesn't fix anything. It is a small feature, so more `minor`

Reply:
I was sure I put minor, I fixed it !